english/code/code.md

```diff
@@ -16,4 +16,6 @@ This also worked!
 
 You then decide to change the transparency program to slowly alternate between 100% and 25%.
 
-Some time passes and an opportunity arises to [follow your destiny.](../destiny/destiny.md) 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.